pkg/shell/base_index.js

```diff
@@ -854,8 +846,10 @@ function CompiledComponents() {
     self.items = {};
 
     self.load = function(manifests, section) {
-        $.each(manifests || { }, function(name, manifest) {
-            $.each(manifest[section] || { }, function(prop, info) {
+        Object.entries(manifests || { }).forEach(m => {
+            const [name, manifest] = m;
```

Comment:
Note that you can use syntactical destructuring to write this in a nicer way: `.forEach(([name, manifest]) => { ...`